CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+2018-06-18
+----------
+
+Changed
+~~~~~~~
+
+* The TAG environment variable is replaced by ST2_IMAGE_TAG.
```

Comment:
Just would be nice to highlight ``TAG`` and ``ST2_IMAGE_TAG`` (instead of TAG and ST2_IMAGE_TAG).